Userland/Applications/MailSettings/MailSettingsWindow.gml

```diff
@@ -0,0 +1,145 @@
+@GUI::Frame {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10, 10, 10, 10]
+        spacing: 5
+    }
+
+    @GUI::GroupBox {
+        title: "Server Settings"
+        fixed_height: 170
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [8, 16, 8, 8]
+            spacing: 2
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 16
+            }
+
+            @GUI::Label {
+                fixed_width: 32
+                fixed_height: 32
+                name: "server_settings_image_label"
+            }
+
+            @GUI::Label {
+                text: "Mail server connectivity settings"
+                text_alignment: "CenterLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 16
+            }
+
+            @GUI::Widget {
+                fixed_width: 32
+            }
+
+            @GUI::Label {
+                text: "Server Address"
+                fixed_width: 80
+                name: "server_label"
+                text_alignment: "CenterLeft"
+            }
+
+            @GUI::TextBox {
+                name: "server_input"
+            }
+        }
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 16
+            }
+
+            @GUI::Widget {
+                fixed_width: 32
+            }
+
+            @GUI::Label {
+                text: "Server Port"
+                fixed_width: 80
+                name: "port_label"
+                text_alignment: "CenterLeft"
+            }
+
+            @GUI::SpinBox {
+                name: "port_input"
+                max: 65535
+            }
+        }
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 16
+            }
+
+            @GUI::Widget {
+                fixed_width: 32
+            }
+
+            @GUI::Label {
+                text: "Use TLS"
+                fixed_width: 80
+                text_alignment: "CenterLeft"
+                name: "tls_label"
+            }
+
+            @GUI::CheckBox {
+                name: "tls_input"
+            }
+        }
+    }
+
+    @GUI::GroupBox {
+        title: "User Settings"
+        fixed_height: 110
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [8, 16, 8, 8]
+            spacing: 2
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 16
+            }
+
+            @GUI::Label {
+                fixed_width: 32
+                fixed_height: 32
+                name: "user_settings_image_label"
+            }
+
+            @GUI::Label {
+                text: "User related settings"
```

Comment:
Same as above, something like "These settings let you identify to the mail recipient."